main.tex

```diff
@@ -16,8 +16,36 @@
 }
 
 \begin{document}
+\section{Facebooks Absicht/Gründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 auf den Markt gebracht und seit da an ist es, und das dahinterstehende Unternehmen, facebook inc, stetig gewachsen. 
+Doch da die jährliche Anzahl neuer Kabelanschlüsse, für das Internet, global gesehen seit 2008 immer weniger geworden ist,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook äquivalent dazu auch nachgelassen.\\
+Dies passiert obwohl "at the beginning of 2016, only an estimated 3.2 billion people — 44 percent of the world’s population are online and connected to the digital economy." (\cite{connectWorld})\\
+Der Grund dafür ist, dass der Großteil der restlichen 56\% aus Menschen besteht, die in den Entwicklungsländern leben. Diese haben entweder nicht die Möglichkeit Internet zu benutzen (wegen einen zu teuren/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da er ihnen nie vermittelt worden ist.
 
-\maketitle
+\medskip
 
+Daraus lässt sich ableiten, dass für den weiteren Wachstum des Internets, und der Dienstleistungen auf ihm (wie zum Beispiel Facebook), es nötig ist dass das Internet in den Entwicklungsländern ausgebreitet, und die Interesse der Menschen dort dafür geweckt wird. 
+
+\medskip
+
+Mark Zuckerberg, Gründer und Vorstandsvorsitzender von Facebook Inc., hat in 2013 das Dokument "Is Connectivity a Human Right?" veröffentlicht.
+In diesem erläutert er, dass es für Facebook wahrscheinlich nicht gewinnbringend sei, das Internet in den Entwicklungsländern auszubreiten. Trotzdem arbeitet Facebook daran dies zu machen, da er daran glaubt, dass jeder Mensch es verdient hat verknüpft zu sein.(Vgl. \cite{HumanRight})
+
+\medskip
+
+Positive Auswirkungen durch eine Ausbreitung des Internets, welches eine Bildungs- und Verbindungsmöglichkeit darstellt, in Ländern der 3. Welt, sind die Entstehung neuer Arbeitsplätze, der Rückgang der Armut und die positive Entwicklung der Wirtschaft.\\
+    
+In dem Report 'Connecting the world - Ten mechanisms for global inclusion', welcher von Facebook zur Unterstützung des Internet.org Projektes in Auftrag gegeben wurden ist, werden die globalen Auswirkungen verdeutlicht:
+                    
+"Based on an econometric analysis of 120 countries, we believe that achieving universal Internet penetration could expand world output by \$6.7 trillion.  
```

Comment:
`Based on an econometric analysis of 120 countries, ` -> `[...]` :)

Reply:
Und korrekte Anführungszeichen in Latex als \`\` statt " (also nur am Anfang des Zitats).
Gilt auch für Zeile 22